pkg/cli/debug/debug.go

```diff
@@ -109,6 +108,10 @@ var (
 		# Debug a node as an administrator
 		oc debug node/master-1
 
+		# Debug a Windows Node
+		# Note: the chosen image must match the Windows Server version (2019, 2022) of the Node
```

Comment:
Is it possible to add a validation for this?

Reply:
Not that I know of